src/index.js

```diff
@@ -28,9 +28,13 @@ export default class Fetch {
     return new Fetch(defaultURL, defaultHeaders, options)
   }
 
-  buidAuthorizationHeader(key) {
+  buildAuthorizationHeader(key) {
     if (key) {
-      const authKeyword = this.options.defaultAuthorizationKeyword || DEFAULT_AUTHORIZATION_KEYWORD
+      const authKeyword = (
```

Comment:
It was like that. But that doesn't you allow to use an empty string `keyword`.. So you gotta explicitly check if the option is set (with `=== undefined`)